react-components/src/components/RevealToolbar/MeasurementButton.tsx

```diff
@@ -39,25 +39,19 @@ export const MeasurementButton = (): ReactElement => {
     measurementTool.exitMeasurementMode();
   };
 
-  const handleMeasurement = (enable: boolean): void => {
+  const handleMeasurement = (_enable: boolean): void => {
     if (viewer.models.length <= 0) {
       return;
     }
     setMeasurementEnabled((prevState) => !prevState);
-    if (enable) {
-      enterMeasurement();
-    } else {
-      exitMeasurement();
-    }
```

Comment:
Why we don't need that part anymore?

Reply:
It's automatically handled by the `useEffect`